src/kepler_model/cmd/cmd_util.py

```diff
@@ -175,10 +176,14 @@ def get_validate_df(data_path, benchmark_filename, query_response):
                     item["total"] = filtered_df[query].max()
                     items += [item]
     energy_queries = [query for query in query_results.keys() if "_joules" in query]
-    print("Energy Queries: ", container_queries)
+    print("Energy Queries: ", energy_queries)
```

Comment:
```suggestion
    logger.debug(f"Energy Queries: {energy_queries} ")
```

Reply:
logger is not imported in model server. Let's have this in another refactoring.